src/meta.jl

```diff
@@ -20,10 +20,10 @@ macro op(f)
         end => opname
     end
     opname === nothing && error("Invalid usage of @op")
-    # opname = f.args[1].args[1]
+
     already_registered = opname ∈ registered_ops
     push!(registered_ops, opname)
-    @assert(isdefined(:tf)) # Need tf as name for module where this code is located
```

Comment:
Why don't we need this anymore? Won't this let a bunch of things break?
Particularly if people start wanted to define their own ops outside this package.
(eg I think I have a `@op Base.atan2(x,y;name=nothing)=....` somewhere in my code)